man/sd_bus_add_match.xml

```diff
@@ -156,9 +156,9 @@
   <refsect1>
     <title>History</title>
     <para><function>sd_bus_add_match()</function> was added in version 231.</para>
-    <para><function>sd_bus_add_match_async()</function> was added in version 237.</para>
-    <para><function>sd_bus_match_signal()</function> was added in version 237.</para>
-    <para><function>sd_bus_match_signal_async()</function> was added in version 237.</para>
+    <para><function>sd_bus_add_match_async()</function>,
+    <function>sd_bus_match_signal()</function> and
```

Comment:
Minor nit: there should probably be the Oxford comma between the penultimate item and *and* if there are three or more items in the list.

Reply:
Added.